web-ui/github.ml

```diff
@@ -49,6 +69,22 @@ let breadcrumbs steps page_title =
     List.rev steps
   )
 
+let statuses ss =
+  let open Tyxml.Html in
+  let render_status (s, elms) =
+    let s_class =
```

Comment:
```suggestion
    let status_class_name =
```
I don't like the similarity with `a_class` which is something completely different.

Reply:
Good idea